addons/account/views/res_config_settings_views.xml

```diff
@@ -375,6 +375,13 @@
                         </block>
                     </app>
                 </xpath>
+                <xpath expr="//block[@name='performance']" position="inside">
+                    <setting id="account_dont_prioritize_invoiced_products" groups="base.group_no_one"
+                             help="Disable prioritizing already invoiced products in bills, purchase orders and sale orders.
+                                   To use in case the computation of products' last invoice date causes performance issue.">
+                        <field name="account_dont_prioritize_invoiced_products"/>
+                    </setting>
+                </xpath>
```

Comment:
It's a bit too soon. We will first check if people have issue with it

Reply:
Ok, I'll keep this commit in my secret pocket then :eyes: